### ISSUE TITLE
glm 遇到 FastLLM Error: Embedding's weight's dim should be 2

### ISSUE THREAD

**Issue by BoFan-tunning** [ztxz16/fastllm#278]
FastLLM Error: Embedding's weight's dim should be 2

**Comment by Mrfranken** [comment]
同样遇到这个问题
`In [2]: import sys
   ...: from peft import PeftModel
   ...: from transformers import AutoModel, AutoTokenizer

In [3]: sys.path.append('..')

In [4]: chatglm2 = '/root/chatglm/m2-6b-chatglm'

In [5]: adapter = '/root/chatglm/ChatGLM-Efficient-Tuning/sleeping_cell'

In [6]: model = AutoModel.from_pretrained(chatglm2, device_map='cpu', trust_remote_code=True)
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████| 7/7 [00:12<00:00,  1.78s/it]

In [7]: model = PeftModel.from_pretrained(model, adapter)

In [8]: model = model.eval()

In [9]: tokenizer = AutoTokenizer.from_pretrained(chatglm2, trust_remote_code=True)

In [10]: from fastllm_pytools import llm

In [11]: model = llm.from_hf(model, tokenizer, dtype = "float16")
convert ( 256 / 256 )
Warmup...
FastLLM Error: Embedding's weight's dim should be 2.
`
